config/example/node.yaml

```diff
@@ -1,159 +1,159 @@
 logger:
-  level: debug
+  level: debug  # logger level: one of "info"(default), "debug", "warn", "error", "dpanic", "panic", "fatal"
```

Comment:
what is the difference between panic and dpanic?

Reply:
lib that we use documents it as following:
```
// DPanicLevel logs are particularly important errors. In development the
// logger panics after writing the message.
    DPanicLevel
// PanicLevel logs a message, then panics.
    PanicLevel
```

DPanic means "development panic", our logger is going to panic only in the development mode

Reply:
> DPanic means "development panic", our logger is going to panic only in the development mode

This is a good note!